antora-ui-camel/src/css/pagination.css

```diff
@@ -29,11 +29,11 @@ nav.pagination span::before {
 }
 
 nav.pagination .prev::before {
```

Comment:
Style changes in this file should be on a separate commit or in a new pull request.